src/richie/apps/courses/models/course.py

```diff
@@ -16,10 +16,10 @@
 from cms.models.pluginmodel import CMSPlugin
 from filer.fields.image import FilerImageField
 
-from ...core.fields.multiselect import MultiSelectField
-from ...core.models import BasePageExtension, PagePluginMixin
 from .category import Category
 from .organization import Organization
+from ...core.fields.multiselect import MultiSelectField
+from ...core.models import BasePageExtension, PagePluginMixin
```

Comment:
I think you get these errors because you forgot to run `make bootstrap` after rebasing to master.